sbi/inference/trainers/npse/npse.py

```diff
@@ -345,12 +358,6 @@ def default_calibration_kernel(x):
             # moving average of the training loss.
             if len(self._summary["training_loss"]) == 0:
                 self._summary["training_loss"].append(train_loss_average)
-            else:
-                previous_loss = self._summary["training_loss"][-1]
-                self._summary["training_loss"].append(
-                    (1.0 - ema_loss_decay) * previous_loss
-                    + ema_loss_decay * train_loss_average
-                )
```

Comment:
why was this removed?

Reply:
This was used to reduce the variance in the loss. In practice, it makes convergence slower. I think it is not needed with the current validation loss.